src/sys/select.rs

```diff
@@ -53,6 +53,20 @@ impl FdSet {
             *bits = 0
         }
     }
+
+    /// Finds the largest file descriptor in the set.
+    ///
+    /// Returns `None` if the set is empty.
+    pub fn largest(&self) -> Option<RawFd> {
```

Comment:
I had to make this function public in order to add a test. For some reason putting a `#[test]` function in this module didn't run the test.